openstax/settings/base.py

```diff
@@ -216,3 +214,32 @@
 
 # WAGTAIL SETTINGS
 WAGTAIL_SITE_NAME = 'openstax'
+
+##################################
+#        ACCOUNTS SETTINGS       #
```

Comment:
I agree with this - the app will work if these settings are in dev.py. I'm voting to remove these from either this file or dev.py - do not leave them in both places.